src/main/kotlin/no/nav/klage/oppgave/domain/kafka/KlageStatistikkTilDVH.kt

```diff
@@ -0,0 +1,136 @@
+package no.nav.klage.oppgave.domain.kafka
+
+import com.fasterxml.jackson.annotation.JsonFormat
+import com.kjetland.jackson.jsonSchema.annotations.JsonSchemaDescription
+import com.kjetland.jackson.jsonSchema.annotations.JsonSchemaFormat
+import com.kjetland.jackson.jsonSchema.annotations.JsonSchemaTitle
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+const val DATE_TIME_FORMAT = "yyyy-MM-dd'T'HH:mm:ss.SSSSSS"
+const val DATE_FORMAT = "yyyy-MM-dd"
+
+/**
+ * Brukes av DVH
+ * KlageKvalitetStatistikk er en hendelse i en Sak, knyttet til en konkret behandlingstype (eks. søknad, revurdering, endring, klage).
+ * Vi sender dette typisk ved mottak, tildeling og fullføring.
+ */
+@JsonSchemaTitle("SaksbehandlingKA")
+data class KlageStatistikkTilDVH(
+
+    @JsonSchemaDescription("Kode som angir hvilken enhet som er ansvarlig for behandlingen på det gjeldende tidspunktet")
+    val ansvarligEnhetKode: String?,
+
+    @JsonSchemaDescription("Kode som angir hvilken type enhetskode det er snakk om, som oftest NORG.")
+    val ansvarligEnhetType: String = "NORG",
+
+    @JsonSchemaDescription("Feltet angir hvem som er avsender av dataene, (navnet på kildesystemet).")
+    val avsender: String = "Kabal",
+
+    @JsonSchemaDescription("Nøkkel til den aktuelle behandling, som kan identifisere den i kildesystemet.")
+    val behandlingId: String?,
+
+    @JsonSchemaDescription("Nøkkel til den aktuelle behandling, som kan identifisere den i Kabal.")
+    val behandlingIdKabal: String,
+
+    @JsonFormat(
+        shape = JsonFormat.Shape.STRING,
+        pattern = DATE_FORMAT
+    )
+    @JsonSchemaFormat(DATE_FORMAT)
+    @JsonSchemaDescription("Når behandlingen startet i KA")
+    val behandlingStartetKA: LocalDate?,
+
+    @JsonSchemaDescription("Kode som angir den aktuelle behandlingens tilstand på gjeldende tidspunkt. Ha med alle mulige statuser som er naturlig for det enkelte system/ytelse. Som minimum, angi om saken har følgende status: Registrert, Klar for behandling, Venter på bruker, venter på ekstern (arbeidsgiver, lege etc.), venter på utland, Avsluttet.Her bør det også angis at saken er behandlet av beslutter, men sendt i retur for ny behandling.")
+    val behandlingStatus: String,
```

Comment:
Er dette basert på et kodeverk eller noe sånt?

Reply:
Nei.. Ikke hva jeg vet. På et tidspunkt (om ikke lenge) så skal jeg/vi presentere dette for DVH, og tipper vi kan få noen innspill når vi ser på "reelle data" da.

Reply:
Alt i denne klassen er String, også der det i "bunn" er en enum. JsonSchema støtter enums, vil ikke det gjøre schema bedre/tydeligere/mer brukervennlig om vi hadde brukt det?

Reply:
Jo, kan være dokumentasjonen blir bedre med det.